### ISSUE TITLE
endergenic generator doesnt render ui 1.16.3 3.0.8

### ISSUE THREAD

**Issue by fred21O4** [McJtyMods/RFToolsPower#58]
the ui that used to appear above the generator in older version when holding the wrench no longer appears

**Comment by QuantumEPR** [comment]
Had the same issue. Screenshots taken from Direwolf20 1.16 pack from FTB team. I have also tried in a fresh install of 1.16.5 with newest forge version 36.0.43 with all the relations added. The endergenic generator does not show any statistics (with or without the smart wrench, shifting does not matter).
![2021-02-27_03 17 53](https://user-images.githubusercontent.com/32603388/109386507-e7e3ad80-78af-11eb-87d6-06778658f1c8.png)

![2021-02-27_03 18 02](https://user-images.githubusercontent.com/32603388/109386506-e619ea00-78af-11eb-9013-e40397658441.png)

![2021-02-27_03 17 56](https://user-images.githubusercontent.com/32603388/109386503-e4e8bd00-78af-11eb-9d8d-db7e6a342730.png)

**Comment by McJty** [comment]
yes, that's not implemented yet. If you sneak then The One Probe will also show similar information

**Comment by QuantumEPR** [comment]
> yes, that's not implemented yet. If you sneak then The One Probe will also show similar information

~~So The One Probe should display statistics for the endergenic generator on sneak? This modpack does have The One Probe but I don't think there was any details displayed. Was that also not implemented yet?~~

Edit: Corrected, The One Probe is working with the stats. Thanks for the information!
